cilium-dbg/cmd/root.go

```diff
@@ -93,14 +93,14 @@ const completionExample = `
 ## or, if running Bash 4.1+
 	brew install bash-completion@2
 ## afterwards you only need to run
-	cilium completion bash > $(brew --prefix)/etc/bash_completion.d/cilium
+	cilium-dbg completion bash > $(brew --prefix)/etc/bash_completion.d/cilium
```

Comment:
Nit: Under which circumstances would someone run cilium-dbg on macOS? :thinking: It's technically possible, just wondering if we encourage this or not.